src/components/leap-elements/LiquidityModal.tsx

```diff
@@ -30,6 +30,8 @@ const chainIdForCollateralPetname = (petname?: string) => {
       return 'cosmoshub-4';
     case 'stTIA':
       return 'stride-1';
+    case 'stkATOM':
+      return 'core-1';
```

Comment:
https://github.com/cosmos/chain-registry/blob/5a2352d22de715e0115fc64887944993bea8e123/persistence/chain.json#L4

Reply:
thanks, I had to do some google searching without this